src/main/java/com/nexters/goalpanzi/config/RedissonConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.nexters.goalpanzi.config;
+
+import org.redisson.Redisson;
+import org.redisson.api.RedissonClient;
+import org.redisson.config.Config;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.Resource;
+
+import java.io.IOException;
+
+@ConditionalOnProperty(name = "spring.data.redis.redisson.enabled", havingValue = "true")
+@Configuration
+public class RedissonConfig {
+
+    @Value("${spring.data.redis.redisson.file}")
+    private Resource redissonConfigFile;
+
+    @Bean
+    public RedissonClient redissonClient() throws IOException {
```

Comment:
이런식으로 하면 따로 레디스 설정파일 안 만들어도 될 듯! 
단순 제안이라 유정이가 취사 선택해 🙏🏻

```suggestion
    private final static String HOST_PREFIX = "redis://";

    @Bean
    public RedissonClient redissonClient(
            RedisProperties redisProperties
    ) {
        Config config = new Config();
        config.useSingleServer()
                .setAddress(HOST_PREFIX + redisProperties.getHost() + ":" + redisProperties.getPort());
        return Redisson.create(config);
    }
```

Reply:
처음에 요렇게 짰었는데 나중에 아래처럼 복잡해지면 이 방식이 더 좋지 않을까 싶어서 리팩토링 했어.
```yml
singleServerConfig:
   address: "rediss://127.0.0.1:6379"
   password: "{aes}M+TfpT4T6psLCfS+RHKT7Fx0j6r5wOX535G3NMnaphY="
   sslTruststore: file:truststore
   sslTruststorePassword: "{aes}31paDOrhnyPfDxXPgqyLZF8QR5yJU3U1bZfhsuM4Ruo="
   secretKey: file:secret_key
```
[https://redisson.org/docs/configuration/#yaml-config](https://redisson.org/docs/configuration/#yaml-config)